app/models/expression_file_info.rb

```diff
@@ -0,0 +1,42 @@
+class ExpressionFileInfo
+  include Mongoid::Document
+
+  embedded_in :study_file
+
+  field :library_construction_protocol, type: String
+  field :units, type: String
+  field :biosample_input_type, type: String
+  field :multimodality, type: String
+  field :is_raw_counts, type: Boolean, default: false
+
+  # note that species and reference genome/annotation live at the study_file level, not here
+
+  UNITS_VALUES = [nil, 'UMI-corrected raw counts', 'raw counts']
+  validates :units, inclusion: {in: UNITS_VALUES}
+
+  BIOSAMPLE_INPUT_TYPE_VALUES = [nil, 'Whole cell', 'single nuclei', 'bulk']
+  validates :biosample_input_type, inclusion: {in: BIOSAMPLE_INPUT_TYPE_VALUES}
+
+  MULTIMODALITY_VALUES = [nil, 'CITE-seq', 'Patch-seq']
+  validates :multimodality, inclusion: {in: MULTIMODALITY_VALUES}
+
+  LIBRARY_CONSTRUCTION_VALUES = [nil,
```

Comment:
Could you add a code comment on why we opt for putting these controlled-lists directly in code, in contrast to the controlled lists in metadata convention that exist as external JSON Schema?

The rationale makes sense to me at the moment, I think: 

1. These fields aren't currently integrated with cell metadata validation
2. Nor are they currently intended for all the search use cases we have for cell metadata

But that will quickly fade from my memory, and without a contextual explanation this point will confuse me when re-visiting the code.

Reply:
That's a great point -- I actually hadn't thought about this deeply yet.  I'll touch base with Jean as to how frequently we expect these lists to change

Reply:
After discussing with Jean, we agreed these lists won't change frequently in the short term, so we can create a separate ticket for designing an SCP controlled-list architecture.